README.md

```diff
@@ -35,19 +28,7 @@ Most of the folder structures are governed by the following environment variable
 
 if you want to capture data from your current mac, please follow the instructions [here](Prerequisites.md)
 
-# Basic setup
```

Comment:
not sure why you want to delete these lines. imo, they set the ground rules / intention about the 2-stage backup - which in turn allows ppl to stop with step 1 if they choose to.